tests/metrics/test-raf.sh

```diff
@@ -46,21 +46,18 @@ echo "👍🏻 Didn't fail in non-git directory"
   git config user.name 'Foo'
   file="temp_file"
   file2="temp_file1"
-  touch "${file}"
+  touch -d "1 second ago" "${file}"
   git add "${file}"
   git config commit.gpgsign false
   git commit --quiet -m "first"
   "${LOCAL}/metrics/raf.sh" "${file}" "t1"
-  sleep 1
-  touch "${file2}"
+  touch -d "1 second ago" "${file2}"
```

Comment:
@Orillio I think that this is not a stable solution, because with "1 second" the test is still flaky. If we put "1 year" instead and then "grep" for only the largest part of the number (like `grep "raf 31536"`), the test will be rather stable

Reply:
Rewrote the test according to your comment. Changed `raf.sh` file, because `file_creation` was pulling wrong date of file creation, if using `touch -d` flag to create the file.